specification/fabric/Microsoft.Fabric.Management/tspconfig.yaml

```diff
@@ -24,15 +24,15 @@ options:
     generate-test: true
     generate-sample: true
   "@azure-tools/typespec-ts":
-    azureSdkForJs: true
+    azure-sdk-for-js: true
```

Comment:
I was told this option is no need now since branding and non-branding sdk are seperated. Could you double confirm? cc @MaryGao